src/js/components/message-indicator.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import DocumentTitle from 'react-document-title';
+
+const BLINKING_INTERVAL = 1500;
+
+export class MessageIndicatorComponent extends Component {
+    state = {
+        initialDocumentTitle: global.document ? document.title : '',
```

Comment:
Should this value be refreshed at some point? If the user switches pages or the page title changes at some point, our value will be stale